tests/integration/test_proxy.py

```diff
@@ -8,14 +8,14 @@
 from contexttimer import Timer
 from hamcrest import assert_that, is_
 
-from mbtest.imposters import Imposter, Proxy
+from mbtest.imposters import Imposter, Proxy, Stub
 from mbtest.matchers import had_request
 
 logger = logging.getLogger(__name__)
 
 
 def test_proxy(mock_server):
-    imposter = Imposter(Proxy(to="http://example.com"))
+    imposter = Imposter(Stub(responses=Proxy(to="http://example.com")))
```

Comment:
Can we do this change in a backwards compatible way?

Reply:
Good point. I've implemented chat.